doc/sphinx/language/core/definitions.rst

```diff
@@ -100,15 +100,16 @@ Section :ref:`typing-rules`.
    If :n:`@reduce` is present then :n:`@ident` is bound to the result of the specified
    computation on :n:`@term`.
 
+   Use the :attr:`sealed` attribute to prevent unfolding the definition.
+   The default is unfoldable.
```

Comment:
I tend to think that it's better for the default to match an attribute, which should still be supported on commands (like locality is, even to do exactly the same as the default). Then this would become:
```suggestion
   The default is :attr:`defined`.
```